client/server.js

```diff
@@ -0,0 +1,15 @@
+const webpack = require('webpack');
+const WebpackDevServer = require('webpack-dev-server');
+const config = require('./webpack.config');
+
+new WebpackDevServer(webpack(config), {
+  publicPath: config.output.publicPath,
+  hot: true,
+  historyApiFallback: true,
+  contentBase: 'public/'
+}).listen(8000, 'localhost', function (error) {
```

Comment:
btw, nice to do (can do it in separate commit, would be adding
```
"config": {
    "port": "1337"
  },
```
added in package.json and use `$npm_package_config_port` env variable so it can be configured via package.json / ENV passed from command line.